focus-gradle-plugin/src/test/kotlin/com/dropbox/focus/FocusPluginTest.kt

```diff
@@ -51,17 +67,31 @@ class FocusPluginTest {
   }
 
   @Test
-  fun singleQuotePath() {
```

Comment:
Instead of replacing this test, can you add a new one? The single-quote test was intentionally introduced as part of resolving #25.